experimental/framework/tinylicious-client/src/TinyliciousAudience.ts

```diff
@@ -0,0 +1,72 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { ServiceAudience } from "@fluid-experimental/fluid-static";
+import { IClient } from "@fluidframework/protocol-definitions";
+import { ITinyliciousAudience, TinyliciousLastEditedResult, TinyliciousMember } from "./interfaces";
+
+export class TinyliciousAudience extends ServiceAudience implements ITinyliciousAudience {
```

Comment:
This is prob fine for now, but I'm always a little hesitant on class inheritance.  Since the service-specific audiences are generally made up of "decorated" members it seems maybe a good candidate to compose a `ServiceAudience` rather than inheriting from it.

Reply:
ServiceAudience would work by default for any Fluid service since `IUser` always contains the user ID. Tinylicious would also provide the user name, and ODSP would further provide the user email.

The thinking here was that developers could extend the ServiceAudience class to include even more additional data to their Member objects beyond what is provided by Fluid. So, for example, if they wanted to add in more details to the IMember object that required making some API requests, they would have the ability to add that into their own implementation of `getMembers`.